index.html

```diff
@@ -909,6 +909,18 @@ <h2>DID Parameters</h2>
 <a data-lt="ascii string">ASCII string</a>.
               </td>
             </tr>
+            <tr>
+              <td>
+<code>subject</code>
+              </td>
+              <td>
+Indicates that the resource to be resolved is the <a>DID Subject</a>, rather
```

Comment:
```suggestion
Indicates that the resource to be dereferenced is the <a>DID Subject</a>, rather
```

Again :)

Reply:
I lost your suggestion when I rebased. Thanks for catching this (again)